### DISCUSSION TITLE
Questions about `subprotocols` crates

### DISCUSSION THREAD

**Discussion by plebhash** [stratum-mining/stratum#1240] [General]
this discussion should guide the documentation of `subprotocols` crates, as described on https://github.com/stratum-mining/stratum/issues/845

**Comment by jbesraa** [comment]
https://stratumprotocol.org/specification/03-Protocol-Overview/#36-common-protocol-messages in here we have `Reconnect` message as part of the common messages protocol, but in the code `Reconnect` lives inside the mining messages crate.

Should probably be moved to `common_messages_sv2` crate.
